dub.sdl

```diff
@@ -0,0 +1,50 @@
+name "dmd"
+description "The DMD compiler"
+authors "Walter Bright"
+copyright "Copyright © 1999-2017, Digital Mars"
```

Comment:
CC @WalterBright @andralex - can we replace/update the copyright notice from Digital Mars to D Language Foundation?